demisto_sdk/commands/validate/tests/tools_test.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from demisto_sdk.commands.validate.tests.test_tools import create_playbook_object
+from demisto_sdk.commands.validate.validators.tools import (
+    collect_all_inputs_from_inputs_section,
+    collect_all_inputs_in_use,
+)
+
+
+@pytest.mark.parametrize(
+    "input_name, expected_result",
+    [
+        (
+            {"0": {"inputs.hello": "test"}, "1": {"inputs.example": "test"}},
+            {"hello: test", "example: test"},
+        ),
+        ({"0": {"inputs": "test"}, "1": {"inputs": "test2"}}, set()),
+    ],
+)
+def test_collect_all_inputs_in_use(input_name, expected_result):
```

Comment:
same here, let's give the playbook as an argument and let's change scenario to case.

Reply:
Done.